cmd/monaco/integrationtest/cleanup.go

```diff
@@ -34,6 +34,7 @@ import (
 // Deletes all configs that end with a test suffix and any Settings created by the given manifest
 func CleanupIntegrationTest(t *testing.T, fs afero.Fs, manifestPath string, loadedManifest manifest.Manifest, suffix string) {
 
+	t.Setenv("MONACO_FEAT_AUTOMATION_RESOURCES", "1") // i don't know why we need to set it here (again)
```

Comment:
just wondering, if this env var isn't set on cleanup anymore, isn't the test suffix one that is used in some names missing as well?

Reply:
nevermind, I just saw the issue was setting the env variable too late :man_facepalming: